backend/src/database/user-content/user-content.service.ts

```diff
@@ -278,6 +278,31 @@ export class UserContentService {
     return aiAnswer != null;
   }
 
+  /**
+   * Counts the number of KI generated answers for a user
+   * MAX number of KI generated answer for not prime user: 15
+   * @param groupID ID of the group from the Question/Discussion the Answer belongs to
+   * @returns number - number of KI generated answers
+   */
+  async countAIAnswersForUser(
+    groupID: string
```

Comment:
If you count the AI answers for one user, is here a group id passed as a parameter? 
The groupId is the connection between answers and a question.
This will not count the ai answers

Reply:
your're right. Thanks